go/vt/vtgate/vindexes/vschema.go

```diff
@@ -1147,6 +1235,10 @@ func (vschema *VSchema) FindRoutedShard(keyspace, shard string) (string, error)
 	return keyspace, nil
 }
 
+func (vschema *VSchema) SetForeignKeyChecks(name string, tbl *Table, keys []*sqlparser.ForeignKeyDefinition) {
+
+}
```

Comment:
Should this be empty?

Reply:
I plan to use this method at some point but did not require it. I will remove it.